tick/optim/solver/src/svrg.cpp

```diff
@@ -1,127 +1,150 @@
 // License: BSD 3 clause
 
-//
-// Created by Martin Bompaire on 23/10/15.
-//
-
+#include <model_labels_features.h>
```

Comment:
include with "" instead of <> (XCode compatibility)

Reply:
Done